webapp/app/static/jsx/transaction/RevealsTimeline.jsx

```diff
@@ -88,13 +89,13 @@ export default function RevealTimeline({ addr, loadNewData, aliases }) {
                             <Bar name={aliases.multi_denomination || 'multi_denomination'} dataKey="multi_denomination" stackId="a" fill="violet" />
                             <Bar name={aliases.linked_transaction || 'linked_transaction'} dataKey="linked_transaction" stackId="a" fill="#82ca9d" />
                             <Bar name={aliases.address_match || 'address_match'} dataKey="address_match" stackId="a" fill="lightblue" />
-                            <Bar name={aliases.torn_mine || 'torn_mine'} dataKey="torn_mine" 
-                            // background={({ name }) => (
-                            //     <div style={{ height: '100%', width: '50px', fill: name === selectedWeek ? '#ee11bb' : '#ff0033' }}></div>
-                            // )}
+                            <Bar name={aliases.torn_mine || 'torn_mine'} dataKey="torn_mine"
+                                // background={({ name }) => (
+                                //     <div style={{ height: '100%', width: '50px', fill: name === selectedWeek ? '#ee11bb' : '#ff0033' }}></div>
+                                // )}
                                 stackId="a" fill="white" />
 
-                            <Legend verticalAlign="bottom" height={5} />
+                            <Legend layout="vertical" verticalAlign="bottom" />
```

Comment:
![image](https://user-images.githubusercontent.com/44105002/149240055-796be9ae-e939-4f6c-9833-b5cbdba6e1f7.png)
 i don't like how the legend overlaps with the graph - could you make sure it doesn't? I think setting the 'height' attribute helps with that (like `height={5}` .

Reply:
Thank you! I solved it using height={100}